app/src/main/java/org/dhis2/data/jira/JiraIssueListResponse.kt

```diff
@@ -0,0 +1,25 @@
+package org.dhis2.data.jira
+
+import com.google.gson.Gson
+import java.io.IOException
+
+data class JiraIssueListResponse(
+    val maxResults: Int,
+    val total: Int,
+    val issues: List<JiraIssue>
+) {
+    companion object {
+        fun fromJson(response: String?): JiraIssueListResponse {
+            return try {
+                Gson()
+                    .fromJson(
+                        response,
+                        JiraIssueListResponse::class.java
+                    )
```

Comment:
Do you think is a good idea to add a ConverterFactory(GsonConverterFactory.create()) to retrofit initialization and avoid this?

Reply:
yes, that would be nice 👍